### PR TITLE
Fix blob stuff

### LINKED ISSUE #7712: readapt strain kicks you of the server as blob
### Environment
- Game Version: Staging 21110213
- OS Version: Arch Linux

### Detailed description of bug

when you press the readapt strain button as a blob you get kicked out of the server

https://user-images.githubusercontent.com/80334192/139925362-e70388d9-9cd8-41b6-a935-ea7934e5d085.mp4

### Steps to reproduce

1. Become blob
2. press readapt
3. get kicked sucker!!

### LINKED ISSUE #7713: blob warning spam
### Environment
- Game Version: Staging 21110213

### Detailed description of bug

When the blob reaches a considerable size the biohazard warning will be shown, problem is that it gets spammed on chat every 30s
blob

### Steps to reproduce

1.spawn as blob 
2. grow big
3. wait for the message

### PR DESCRIPTION
**Pull request by ThatDan123**
Fixes #7712
Fixes #7713
CL: [Fix] fixed blob readapt strain kicking the player from the server.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs [hunk 1]
```diff
@@ -18,9 +18,6 @@ public class BodyFat : BodyPartFunctionality, IMovementEffect
 
 		public float CrawlingSpeedModifier { get; private set; }
 
-
-		public Stomach RelatedStomach;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs:22]: why does this need to be removed to fix blob?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs:22]: > why does this need to be removed to fix blob?

It's null as it isnt set anywhere and isnt used so i just removed it

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs:22]: Also causes NRE for blob spawn as it gibs, so this currently breaks all gibbing

Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs:22]: why do Blobs have BodyFat?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs:22]: it should be set when spawned via stomach

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs:22]: > why do Blobs have BodyFat?

they dont, the player they gib out of does

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs:22]: yeah  
var Added = Spawn.ServerPrefab(BodyFatToInstantiate.gameObject).GameObject.GetComponent<BodyFat>();
				BodyFats.Add(Added);
				
			var Added = Spawn.ServerPrefab(BodyFatToInstantiate.gameObject).GameObject.GetComponent<BodyFat>();
				Added.SetAbsorbedAmount(0);
needs to be set  in those two places in stomach

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs:22]: done

**Review by Bod9001 — APPROVED** [review]

**Review by corp-0 — APPROVED** [review]

**Pull request merged**